upgrade/UPGRADE-v11.0.0-dev.md

```diff
@@ -220,3 +220,4 @@ There you can find links to upgrade notes for other versions too.
                 )
             ```
     - method `Shopsys\ReadModelBundle\Image\ImageViewFacade::getForEntityIds()` was removed, use `getMainImagesByEntityIds()` instead
+    - method `Shopsys\ReadModelBundle\Product\Action\ProductActionViewFacade::getForArray()` was removed, use `ProductActionViewFactory` instead
```

Comment:
I would `::createFromArray()` here :slightly_smiling_face:

Reply:
added in 89bb0f956